docs/pages/desktop-access/troubleshooting.mdx

```diff
@@ -187,3 +170,69 @@ issue, you can
 your own third party certificate. Note that Active Directory is
 [extremely picky](https://docs.microsoft.com/en-us/troubleshoot/windows-server/identity/enable-ldap-over-ssl-3rd-certification-authority#requirements-for-an-ldaps-certificate)
 so take care to generate your certificates correctly.
+
+## Desktops are not discovered via LDAP
+
+### LDAP not yet initialized
+
+Teleport is running, but desktops do not show up in the web UI. The logs contain
+errors similar to:
+
+```
+skipping desktop discovery: LDAP not yet initialized
+```
+
+**Solution:** Confirm Teleport certificate is installed
+
+Teleport uses a Teleport-issued certificate to authenticate with the LDAP
+server. This error occurs when Teleport unable to authenticate, which is
+often due to its certificate authority not being trusted by Active Directory.
+
+First, verify that the Teleport CA is present in the LDAP NTAuthStore. Run the
```

Comment:
Should we capitalize "NTAuthStore" the way this doc does it? https://docs.microsoft.com/en-us/troubleshoot/windows-server/windows-security/import-third-party-ca-to-enterprise-ntauth-store